conda.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: chemnlp
 dependencies:
-    - python>=3.7
```

Comment:
Didn't you say you wanted to pin it? Then we can maybe also do it here?

Reply:
I think you've commented on the original file -> I've bumped to 3.8? I think we could go to 3.9 but would perhaps leave this to a new PR.